CONTRIBUTING.md

```diff
@@ -30,6 +30,18 @@ These steps outline the general contribution workflow:
 * Make sure the tests pass, and add any new tests as appropriate.
 * Submit a pull request to the original repository. (see [Format of Pull Requests](#format-of-pull-requests))
 
+## Required tools
+To allow scripts and `make` targets working correctly, ensure you have following tools installed in your system:
+
+* golang (see `go.mod` file for minimum required go version)
+* `awk` (GNU variant)
+* `sed` (GNU variant)
+* `make`
+* `curl`
+* python and pyyaml package
```

Comment:
you might want to call out that the tooling is currently looking for python2, this caused some surprise for me when trying to install pyyaml.

Reply:
good point! @paulfantom

Reply:
Added.